coops-ui/tests/unit/App.spec.ts

```diff
@@ -191,13 +191,15 @@ describe('App', () => {
               'streetAddress': '1012 Douglas St',
               'addressCity': 'Victoria',
               'addressRegion': 'BC',
+              'addressType': 'mailing',
```

Comment:
Nice test... Please add a comment here or in expects below to say what you're testing. I understand this because we discussed this, but to another reviewer this will look like an error. Thx.

Reply:
ahh, i see. Will do!

Reply:
Done.